duckdb-vortex/src/vortex_extension.cpp

```diff
@@ -53,7 +53,7 @@ struct VortexBindData : public TableFunctionData {
 struct VortexScanLocalState : public LocalTableFunctionState {
 	idx_t current_row = 0;
```

Comment:
Or we could do this the other way around like for `GlobalState` that we rely on default initialization. Probably better as more consistent. I was just double checking here whether this is always properly initialized.

wdyt? @joseph-isaacs